workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFClaimForm/components/MDFClaimPage/MDFClaimPage.tsx

```diff
@@ -90,40 +90,30 @@ const MDFClaimPage = ({
 		return isDisplayable;
 	};
 
+	const availableMDFActivities = values.activities?.filter((activity) =>
+		isDisplayableMDFActivityClaim(activity)
+	).length;
+
+	const getCreateClaimDenialMessage = () => {
```

Comment:
Make sure to sort variables alphabetically

Reply:
Hey, Ryan! I did a minor change in a IF statement, but i couldn't figure out which variables could be sorted without change code integrity.

Reply:
I see, order matters in this case, and there is an extra line break between them, so that if fine. In general, if order does not matter, this is no blank line and its alphabetized, if order does matter (as is the case here) then add the line break and it makes sense. Thanks for the explanation